ee/debug/checkups/binary_directory.go

```diff
@@ -68,10 +71,46 @@ func getBinDir() string {
 	case "darwin":
 		return "/usr/local/kolide-k2"
 	case "linux":
+		if allowedcmd.IsNixOS() {
```

Comment:
```suggestion
	case "darwin", "linux":
		if allowedcmd.IsNixOS() {
```

I think?

Reply:
I don't think so -- `IsNixOS` is an OS check, not if the package manager is in use. I kept it pretty narrowly scoped to start.

I am not sure how much of the changes I've made so far adequately support Nix on Darwin right now -- could be fine, but I haven't tested it at all yet.